src/components/ebay-pagination/README.md

```diff
@@ -0,0 +1,56 @@
+# ebay-pagination
+
+The `<ebay-pagination>` is a tag used to create a pagination navigation.
+
+
+## ebay-pagination Tag
+
+### ebay-pagination Usage
+
+```marko
+<ebay-pagination aria-label-prev="Previous page" aria-label-next="Next page" curr-text="Results Pagination - Page 1">
+    <ebay-pagination-item href="#" previous disabled/>
+    <ebay-pagination-item href="#">item 1</ebay-pagination-item>
+    <ebay-pagination-item href="#" current>item 2</ebay-pagination-item>
+    <ebay-pagination-item href="#">item 3</ebay-pagination-item>
+    <ebay-pagination-item href="#" next/>
+</ebay-pagination>
+```
+
+### ebay-pagination Attributes
+
+Name | Type | Stateful | Description
+--- | --- | --- | ---
+`class` | String | No | custom class
+`aria-label-prev` | String | No | aria-label for previous button
+`aria-label-next` | String | No | aria-label for previous button
+`curr-text` | String | No | Description for the current page (e.g. Results of Page 1)
+`fake-link` | Boolean | No | Use pagination links for an ajax reload
+
+### ebay-pagination Events
+
+Event | Data | Description
+--- | --- | ---
+`pagination-previous` |  `{ event, el: event.target }`| clicked previous arrow
+`pagination-next` | `{ event, el: event.target }` | clicked next arrow
```

Comment:
(minor) `button` instead of `arrow` for consistency

Reply:
I didn't want to say button as they aren't technically a `button` tag, so it might cause confusion

Reply:
Ok, I think just keep consistency with whatever terminology you use. This stood out because for the attributes it has "previous button" and here it has "previous arrow."